tailor.config.js.example

```diff
@@ -0,0 +1,41 @@
+'use strict';
```

Comment:
It would be awesome if this would be a working / real example 🤡

Reply:
What am I missing? 😕 I tried it locally and in fact it does work... 🤔